Description: Put all orphan files into the airportconfig package
 Some files were not declared as being part of a package.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/airportconfig/AboutDialog.java
===================================================================
--- airport-utils-2.orig/airportconfig/AboutDialog.java	2010-03-11 20:55:56.223753672 +0100
+++ airport-utils-2/airportconfig/AboutDialog.java	2010-03-11 20:56:29.665753154 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportconfig;
 
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/airportconfig/AirportBaseStation128BitConfigurator.java
===================================================================
--- airport-utils-2.orig/airportconfig/AirportBaseStation128BitConfigurator.java	2010-03-11 20:55:56.232753515 +0100
+++ airport-utils-2/airportconfig/AirportBaseStation128BitConfigurator.java	2010-03-11 20:56:29.678445530 +0100
@@ -20,7 +20,7 @@
  */
  
 
-
+package airportconfig;
 
 import java.util.*;
 import java.net.*;
Index: airport-utils-2/airportconfig/AirportBaseStationConfigurator.java
===================================================================
--- airport-utils-2.orig/airportconfig/AirportBaseStationConfigurator.java	2010-03-11 20:55:56.240753544 +0100
+++ airport-utils-2/airportconfig/AirportBaseStationConfigurator.java	2010-03-11 20:56:29.695201671 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airportconfig;
 
 
 import java.util.*;
Index: airport-utils-2/airportconfig/AirportBaseStationConfiguratorMicro.java
===================================================================
--- airport-utils-2.orig/airportconfig/AirportBaseStationConfiguratorMicro.java	2010-03-11 20:55:56.249753202 +0100
+++ airport-utils-2/airportconfig/AirportBaseStationConfiguratorMicro.java	2010-03-11 20:56:29.736945943 +0100
@@ -19,7 +19,7 @@
  *
  */
  
-
+package airportconfig;
 
 
 import java.util.*;
Index: airport-utils-2/airportconfig/AirportBaseStationMultiConfigurator.java
===================================================================
--- airport-utils-2.orig/airportconfig/AirportBaseStationMultiConfigurator.java	2010-03-11 20:55:56.261753441 +0100
+++ airport-utils-2/airportconfig/AirportBaseStationMultiConfigurator.java	2010-03-11 20:56:29.753751996 +0100
@@ -20,7 +20,7 @@
  */
  
 
-
+package airportconfig;
 
 import java.util.*;
 import java.net.*;
